Sources/Models/DynamoDBUser.swift

```diff
@@ -3,8 +3,8 @@ import Foundation
 public struct DynamoDBUser: Sendable, Codable {
```

Comment:
You can't change this struct because it's used for decoding from database.
You can only change `DynamoUser`, not `DynamoDBUser`, until we migrate the db.